CardEngine/src/EventBus/abstractEventBase.cpp

```diff
@@ -0,0 +1,5 @@
+﻿#include "pch.h"
```

Comment:
Use ifndef https://google.github.io/styleguide/cppguide.html#The__define_Guard is a good practice